lib/rubocop/cop/layout/multiline_method_call_indentation.rb

```diff
@@ -214,6 +214,18 @@ def operator_rhs?(node, receiver)
           node.operator_method? && node.arguments? &&
             within_node?(receiver, node.first_argument)
         end
+
+        def relative_indentation(lhs, node)
+          indentation = indentation(lhs) + correct_indentation(node)
+          indentation += indentation(node) if node_indentationed?(lhs, node)
```

Comment:
Adding two indentations can't be correct in general.

Here is an example where it does not work:
```ruby
module Foo
  class Bar
    def foo
      params[:key] =
        foo
        .bar
        .baz
    end
  end
end
```
autocorrects to
```
7352.rb:6:9: C: [Corrected] Layout/MultilineMethodCallIndentation: Use 2 (not 2) spaces for indenting an expression in an assignment spanning multiple lines.
        .bar
        ^^^^
7352.rb:7:9: C: [Corrected] Layout/MultilineMethodCallIndentation: Use 2 (not 2) spaces for indenting an expression in an assignment spanning multiple lines.
        .baz
        ^^^^

1 file inspected, 5 offenses detected, 4 offenses corrected
====================
# frozen_string_literal: true

module Foo
  class Bar
    def foo
      params[:key] =
        foo
                .bar
                .baz
    end
  end
end
```
```

Reply:
Thank you for your feedback. It helped me to rethink the issue.

Reply:
Outdated